### PR TITLE
Flask webserver

### PR DESCRIPTION
**Pull request by leondaniel22**

### CODE REVIEW

File: src/web-server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM  python:3
+RUN mkdir WORK_REPO
+WORKDIR /WORK_REPO
+
+# add the directory res to docker image
+ADD res .
+
+COPY . /res
+
+# copy the requiremnts.txt file and install it in the dopcker container
```
Comment (motrell) [inline src/web-server/Dockerfile:10]:
```suggestion
# copy the requiremnts.txt file and install it in the docker container
```

Just a typo

**Review by motrell — APPROVED** [review]
Looks good!

File: src/web-server/Dockerfile (hunk 1 above)
Comment (Amir-Hussein-OTH) [reply src/web-server/Dockerfile]: Done

**Comment by JMiltner97** [comment]
Looks good so far, we'll implement some further adjustments before merging it

**Comment by JMiltner97** [comment]
I think this is obsolete with the pull of nice-looking-website

**Comment by martin-mtv** [comment]
resolved merge conflicts

**Pull request merged**